stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/ChatUI.kt

```diff
@@ -152,4 +153,13 @@ public object ChatUI {
      */
     @JvmStatic
     public var videoThumbnailsEnabled: Boolean = true
+
+    /**
+     * Sets the strategy for resizing images hosted on Stream's CDN. Disabled by default,
+     * set [StreamCdnImageResizing.imageResizingEnabled] to true if you wish to enable resizing images. Note that
+     * resizing applies only to images hosted on Stream's CDN which contain the original width (ow) and height (oh)
+     * query parameters.
+     */
+    @JvmStatic
+    public var StreamCdnImageResizing: StreamCdnImageResizing = StreamCdnImageResizing()
```

Comment:
Also - rather than just creating a value of `StreamCdnImageResizing()` and it having default values, how about having that class w/o default arguments and having an object or factory function `DefaultStreamCdnImageResizing()` that has these values.

Reply:
We can use that approach.

Reply:
Suggestion implemented.